content/news/2024-06-30-stem-support.md

```diff
@@ -0,0 +1,34 @@
+title: "GSoC 2024: Adding STEM mixing support"
+authors: Antoine Colombier
+tags: gsoc, gsoc-2024, improvements
+slug: STEM-Mixing
+date: 2024-06-30 12:00:00
+
+Disclaimer: *The blog post primarily serves as the documentation for the [Google Summer of Code](https://summerofcode.withgoogle.com/) 2024 project: "Multi channel mixing support (STEMS)".*
+
+## Explore New Creative Horizons with Mixxx's Latest Feature: Stem Mixing
+
+We are excited to introduce a major new feature in Mixxx: Stem Mixing. This innovative tool, inspired by Native Instruments' concept from their Traktor Pro software, allows you to separate and manipulate different elements of a track in real-time. But what exactly are stems?
+
+Stems are individual audio tracks that make up a complete song. Typically, a song is split into four main stems: drums, bassline, harmony, and vocals. By isolating these elements, you can remix, mashup, and edit tracks on the fly, providing a new level of flexibility and creativity in your DJ sets.
+
+Mixxx now supports Native Instruments stem files, the current public specification for this format. Whether you're an amateur DJ eager to experiment with new techniques or a professional looking to enhance your performances, Stem Mixing in Mixxx offers powerful new capabilities to elevate your mixes.
```

Comment:
The later part sounds more like an advertise and not like a project report. Maybe you can add something like: Advertised as ...